### PR TITLE
Hud10837/raster layer file kotlin

### PR DESCRIPTION
**Pull request by hud10837**
Another tiny kotlin conversion! This one turned out very small because I've completely removed the permission requests at runtime due to the scoped storage. Main concern is the offline data section in the readme, which I tried to shorten up.

@Rachael-E can you do a first review? Thanks!

### CODE REVIEW

File: kotlin/build.gradle [hunk 1]
```diff
@@ -10,8 +10,7 @@ buildscript {
     }
     dependencies {
         classpath "com.android.tools.build:gradle:$gradleVersion"
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
-    }
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"}
```
[the remark this answers is not in the document]
Comment (hud10837) [reply kotlin/build.gradle]: It's happened so many times I can never remember which is correct! Is it mean to be below the current line, or one space over?

File: kotlin/raster-layer-file/src/main/java/com/esri/arcgisruntime/sample/rasterlayerfile/MainActivity.kt [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    val raster =
+      Raster(getExternalFilesDir(null)?.path + getString(R.string.raster_file_path))
+    // create a raster layer
+    val rasterLayer = RasterLayer(raster)
+    // add the raster as an operational layer
+    map.operationalLayers.add(rasterLayer)
+    // set viewpoint on the raster
+    rasterLayer.addDoneLoadingListener {
+      if (rasterLayer.loadStatus == LoadStatus.LOADED) {
+        mapView.setViewpointGeometryAsync(
```
[the remark this answers is not in the document]
Comment (hud10837) [reply kotlin/raster-layer-file/src/main/java/com/esri/arcgisruntime/sample/rasterlayerfile/MainActivity.kt]: I need to see that PR! I find it the same either way, but multi line is a little bit easier to read for me in some cases. Though just now I find it a bit easier to read on one line. Is there a standard here? I'll go look for that PR

File: kotlin/build.gradle (hunk 1 above)
[the remark this answers is not in the document]
Comment (Rachael-E) [reply kotlin/build.gradle]: below the current line! haha don't worry I got your back!

[the remark this answers is not in the document]
Comment (hud10837) [reply kotlin/build.gradle]: I'm so confused, I pulled it and it was already fixed?? gradle will break me

**Comment by hud10837** [comment]
@Rachael-E that's everything changed now, thanks for that! Can you take another look?

**Comment by hud10837** [comment]
Thanks for that! @sorenoid can you take a second look?

**Pull request merged**